README.md

```diff
@@ -10,7 +10,9 @@
 
 ## 2025
 
+- 2025-06-01: [CBS Sunday Morning: Interview with David Pogue](https://www.cbs.com/shows/video/QvwYk_EGpqRHONqMsKR_2XjJcVJWalC8/)
```

Comment:
Use this link instead https://www.youtube.com/watch?v=ey1rpNtRADg&ab_channel=CBSSundayMorning